tests/vppoutputencode.cpp

```diff
@@ -160,13 +162,37 @@ bool VppOutputEncode::config(NativeDisplay& nativeDisplay, const EncodeParams* e
     m_encoder->setNativeDisplay(&nativeDisplay);
     m_mime = m_output->getMimeType();
     setEncodeParam(m_encoder, m_width, m_height, encParam, m_mime);
-
+    enableLatency = encParam->enableFrameLatency;
     Encode_Status status = m_encoder->start();
     assert(status == ENCODE_SUCCESS);
     initOuputBuffer();
     return true;
 }
 
+void VppOutputEncode::getFrameDelay(VideoEncOutputBuffer m_outputBuffer){
+    struct timeval end;
+    gettimeofday(&end,NULL);
+    int64_t now = 1000000 * end.tv_sec + end.tv_usec;
+    int64_t delay = now - m_outputBuffer.timeStamp;
+    FrameCount++;
+    if (FrameCount == 1){
+        MaxLatency = delay;
+        MinLatency = delay;
+        AveLatency = delay;
+    }
+    else{
+        if (delay > MaxLatency) 
+            MaxLatency = delay;
+        if (delay < MinLatency)
+            MinLatency = delay;
+        AveLatency = (AveLatency * (FrameCount - 1) + delay) / FrameCount;
```

Comment:
if we only have a sum but no frame count, we still can't get the average latency